frontend/src/utilities/const.ts

```diff
@@ -6,6 +6,9 @@ const POLL_INTERVAL = process.env.POLL_INTERVAL ? parseInt(process.env.POLL_INTE
 const FAST_POLL_INTERVAL = process.env.FAST_POLL_INTERVAL
   ? parseInt(process.env.FAST_POLL_INTERVAL)
   : 3000;
+const POLL_INTERVAL_NOTEBOOK_EVENTS = process.env.POLL_INTERVAL_NOTEBOOK_EVENTS
```

Comment:
Not sure you needed to do this -- but I don't see an inherent harm in making another env var